features/admin.applications.v1/pages/applications.tsx

```diff
@@ -176,35 +184,64 @@ const ApplicationsPage: FunctionComponent<ApplicationsPageInterface> = (
         error: myAccountStatusFetchRequestError
     } = useMyAccountStatus(!isSubOrg && applicationConfig?.advancedConfigurations?.showMyAccountStatus);
 
+    /**
+     * Set the application id for My Account.
+     */
+    useEffect(() => {
+        if (myAccountApplicationData?.applications?.length === 1) {
+            myAccountApplicationData.applications.forEach((
+                item: ApplicationListItemInterface
+            ) => {
+                setMyAccountAppId(item?.id);
+            });
```

Comment:
Since applications length is 1, can't we simplify this as follows?

```suggestion
            setMyAccountAppId(myAccountApplicationData.applications[0]?.id)
```

Reply:
+1 to improve. Will improve in a follow up PR.